src/ui/reducers/app.ts

```diff
@@ -60,6 +61,9 @@ const appSlice = createSlice({
   name: "app",
   initialState: initialAppState,
   reducers: {
+    replayAssistToggled(state) {
+      state.showReplayAssist = !state.showReplayAssist;
+    },
```

Comment:
For simple actions like this– ones that only update one piece of state– the name of an action generally _matches_ the name of the state.

So I think this should be either `setShowReplayAssist` or `toggleShowReplayAssist`

Reply:
In this case, I was trying to align to @markerikson 's suggestion for these items to be human-readable while reading through a series of steps.

But he don't *force* me to change it, and I understand it's a matter of style/taste/preference. CC'ing Mark to discuss the best way forward here, I have no opinions but wanted to call out the question.

Reply:
Ah, I see.

I find it kind of confusing to have Redux state, React component state, and Redux actions all with slightly different naming variations. It's more mental overhead of things you have to keep track of while working in the code.

Reply:
I'm going to close this out as a won't fix for now, but I want to stress that I'm totally fine to change this!